tests/baselines/reference/checkJsxChildrenProperty4.errors.txt

```diff
@@ -1,8 +1,6 @@
 tests/cases/conformance/jsx/file.tsx(24,28): error TS2551: Property 'NAme' does not exist on type 'IUser'. Did you mean 'Name'?
 tests/cases/conformance/jsx/file.tsx(36,15): error TS2322: Type '(user: IUser) => Element' is not assignable to type 'string | number | boolean | any[] | ReactElement<any>'.

```

Comment:
Wasn't there supposed to be some new text about "did you mean to call?"?  Or does that not apply in this case?